l3build-aux.lua

```diff
@@ -29,9 +29,11 @@ local match = string.match
 local pairs = pairs
 local print = print
 
-local lookup = kpse.lookup
+local os_time = os.time
+
+---@type l3build_t
+local l3build = require("l3build")
```

Comment:
Access to the shared data.

One could prefer `local l3b = require("l3build")` but in that case, one would also expect
`l3b.arguments = require("l3b.arguments")` which is really borderline.